database/factories/TaxFactory.php

```diff
@@ -12,30 +12,20 @@
 
 class TaxFactory extends Factory
 {
-    /**
-     * The name of the factory's corresponding model.
-     *
-     * @var string
-     */
     protected $model = Tax::class;
 
-    /**
-     * Define the model's default state.
-     *
-     * @return array
-     */
     public function definition()
     {
         $sentence = $this->faker->sentence;
 
         return [
-            'slug'       => Str::slug($sentence),
-            'name'       => $sentence,
-            'title'      => $sentence,
-            'percent'    => $this->faker->numberBetween(1, 50),
-            'tax_code'   => $this->faker->randomElement(TaxCode::getValues()),
-            'location_id'=> Location::factory()->create(),
-            'creator_id' => randomOrCreate(app('user')),
+            'slug'          => Str::slug($sentence),
+            'name'          => $sentence,
+            'title'         => $sentence,
+            'percent'       => $this->faker->numberBetween(1, 50),
+            'tax_code'      => $this->faker->randomElement(TaxCode::getValues()),
+            'location_id'   => Location::factory()->create(),
```

Comment:
Since this field has a unique requirement as there can only be one Tax (#44) entry per Location (https://github.com/tipoff/locations/issues/42), the easiest way to handle it is to create a new Location.